build/cockroach.rb

```diff
@@ -30,11 +30,50 @@ def install
     system "xcrun", "make", "GOFLAGS=-v", "-C",
            "src/github.com/cockroachdb/cockroach", "build"
     bin.install "src/github.com/cockroachdb/cockroach/cockroach" => "cockroach"
+
+    # TODO(pmattis): Debug the launchctl stuff
+    # (prefix+'com.cockroachlabs.cockroachdb.plist').write startup_plist
+    # (prefix+'com.cockroachlabs.cockroachdb.plist').chmod 0644
   end
 
   test do
     system "#{bin}/cockroach", "version"
   end
 
-  # TODO(peter): Need to figure out how set cockroach to running.
+  def caveats
+    <<-EOS.undent
+    Start cockroach:
+        cockroach start --store=/usr/local/cockroach-data
```

Comment:
Best I can think of is `brew audit <name> --strict --online`.

On Wed, Mar 30, 2016 at 10:13 AM, Peter Mattis <notifications@github.com>
wrote:

> In build/cockroach.rb
> <https://github.com/cockroachdb/cockroach/pull/5707#discussion_r57894950>:
>
> >    end
> >
> >    test do
> >      system "#{bin}/cockroach", "version"
> >    end
> >
> > -  # TODO(peter): Need to figure out how set cockroach to running.
> > +  def caveats
> > +    <<-EOS.undent
> > +    Start cockroach:
> > +        cockroach start --store=/usr/local/cockroach-data
>
> Ok. Is there a way to test this without running the brew install again
> (which takes many minutes)?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/5707/files/ff095019f59858b9a885044a7f19d23bd022c1ce#r57894950>
>